students/kosarzewski_maciej/lesson_05_lists/character_picture_gird.py

```diff
@@ -0,0 +1,14 @@
+grid = [['.', '.', '.', '.', '.', '.'],
+        ['.', 'O', 'O', '.', '.', '.'],
+        ['O', 'O', 'O', 'O', '.', '.'],
+        ['O', 'O', 'O', 'O', 'O', '.'],
+        ['.', 'O', 'O', 'O', 'O', 'O'],
+        ['O', 'O', 'O', 'O', 'O', '.'],
+        ['O', 'O', 'O', 'O', '.', '.'],
+        ['.', 'O', 'O', '.', '.', '.'],
+        ['.', '.', '.', '.', '.', '.']]
+
+for y in range(0, 6):
```

Comment:
Try to avoid adding magic numbers, you might use range(len(grid[0]))
By default range method start from '0' so you can omit the first argument with '0'